keps/sig-storage/4049-storage-capacity-scoring-of-nodes-for-dynamic-provisioning/README.md

```diff
@@ -468,6 +490,10 @@ func (pl *VolumeBinding) Score(ctx context.Context, cs *framework.CycleState, po
 }
 ```
 
+### Feature Gate Consolidation
+
+The `StorageCapacityScoring` feature gate will now control the functionality previously managed by the `VolumeCapacityPriority` feature gate, which will be removed. This consolidation focuses on enabling node scoring based on storage capacity, limited to the behaviors necessary for `StorageCapacityScoring`. Any additional functionalities from `VolumeCapacityPriority` that are not required will not be supported. For more details on the original proposal, see [KEP-1845](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1845-prioritization-on-volume-capacity/README.md).
```

Comment:
I think the `VolumeCapacityPriority` feature needs to be deprecated first. 
https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior